tests/cover/test_testdecorators.py

```diff
@@ -203,7 +203,7 @@ def this_has_a_unique_name_and_lives_on_a_class(self, x):
     ) in e.value.args[0]
 
 
-@given(lists(integers()), integers())
+@given(lists(integers(), unique=True), integers())
 def test_removing_an_element_from_a_unique_list(xs, y):
```

Comment:
I don't follow what this is actually testing - it just seems both obviously correct and triply redundant.  Do we still have full coverage if it's deleted?

*Having read further, I think it's just needless contrast with the next test and should be deleted.*

Reply:
The entirety of `test_testdecorators.py` is a bit weird and it's more of a smoke test module than anything very interesting in its own right. I'd rather just leave it alone as much as possible in this PR.